content/efcore/getting-started/adding-ef-core.md

```diff
@@ -0,0 +1,99 @@
+| IBM Data Servers | [EntityFramework.IBMDataServer -Pre](https://www.nuget.org/packages/EntityFramework.IBMDataServer) |
+| InMemory (for Testing) | [Microsoft.EntityFrameworkCore.InMemory](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.InMemory/) |
+| Devart (MySQL, Oracle, PostgreSQL, SQLite, DB2, SQL Server, and more) | N/A, for installation instructions, follow the [Devart dotConnect documentation](https://www.devart.com/dotconnect/). |
+
+##Create an EF Core Project
+
+To create a new .NET Core project using command line: 
+
+1. Create a new directory for the project.  
+
+	mkdir MyProject 
+	
+2. Navigate to this project directory. 
+
+	cd MyProject 
+	
+3. Initialize a .NET Core project (this creates project.json and Startup.cs) 
+
+	dotnet new 
+
+4. Restore dependencies. 
+
+	dotnet restore 
+
+5. Run the project 
+
+	dotnet run
+
+You can verify that Entity Framework Core is installed by running
+
+    dotnet ef -help
+	
+##Using a Database
+###Code-first vs. Database-first
+
+Now that you have a working .NET Core project, we can connect the code project to a database. EF Core allows developers to generate a database from C# code, or to connect an existing database to the project. These methods are known as "code-first" and "database-first." Code-first projects create a database from models made in-code that represent database tables. Developers can populate tables, delete fields, and define keys all from the code without writing a single SQL script. While developers can still modify tables and data from code, database-first projects connect a EF Core project to an existing database. This works well for developers who already have a database in development. 
+
+In this tutorial, we will demonstrate how to generate a database with a code-first project and how to connect a database for database-first. 
+
+###Code-first: Generating a Database
+
+For both code-first and database-first projects, we must set up the database context. Contexts "point" EF Core to the specified database (*.db) file. DbContext is an existing .NET Core class that can be extended to your context. In this example, we are generating a SQLite database from our code. 
+
+If we had the classes "Blog" and "Post" in our project that we want to refer to database tables we would include them in the context (in the later section "Understanding Models" we will discuss these type of classes further).  With these classes, the context would look like: 
+
+	public class BloggingContext : DbContext  
+	{         
+		public DbSet<Blog> Blogs { get; set; } 
+		public DbSet<Post> Posts { get; set; } 
+			
+		protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)   
+		{  
+			optionsBuilder.UseSqlite("Filename=./blog.db");  
+		} 
+	} 
+
+###Database-first: Connecting to a Database
+
+If you are creating a .NET Core application around an existing database, you will probably need a connection string. 
```

Comment:
You need a connection string for both code-first and database-first.

The main difference between code-first and database-first is that code-first uses migrations to create a database.